Updater.php

```diff
@@ -11,74 +11,112 @@
+				'plugin'           => function () use ( $plugin_basename ) {
+					return new Plugin( WP_PLUGIN_DIR . '/' . $plugin_basename );
+				},
+				'get_release_data' => function ( Container $c ) {
+					$cache_key = str_replace( '-', '_', $c['plugin']->slug() ) . '_github_api_latest_release';
+					$payload   = get_transient( $cache_key );
+					if ( ! $payload ) {
+						$payload = new \stdClass();
+						$package_info = array(
 							'vendorName'     => $c['vendor'],
 							'packageName'    => $c['package'],
-							'pluginBasename' => $c['plugin_basename']
-						], null, '&' );
-					$response = wp_remote_get( 'https://bluehost-wp-release.com/v1/' . $query_string );
-					if ( 200 === wp_remote_retrieve_response_code( $response ) ) {
-						$body = wp_remote_retrieve_body( $response );
-						if ( $body ) {
-							$data = json_decode( $body );
-							if ( $data ) {
-								$payload = $data;
-								set_transient( $cache_key, $payload, HOUR_IN_SECONDS * 6 );
+							'pluginBasename' => $c['plugin_basename'],
+						);
+						$query_string = '?' . http_build_query( $package_info, null, '&' );
+						$response     = wp_remote_get( 'https://bluehost-wp-release.com/v1/' . $query_string );
+						if ( 200 === wp_remote_retrieve_response_code( $response ) ) {
+							$body = wp_remote_retrieve_body( $response );
+							if ( $body ) {
+								$data = json_decode( $body );
+								if ( $data ) {
+									$payload = $data;
+									set_transient( $cache_key, $payload, HOUR_IN_SECONDS * 6 );
+								}
 							}
 						}
 					}
-				}
-
-				return $payload;
-			},
-		] );
 
-		add_filter( 'site_transient_update_plugins', function ( $transient ) use ( $container ) {
+					return $payload;
+				},
+			)
+		);
 
-			$plugin = $container['plugin'];
-			$release = $container['get_release_data'];
+		add_filter(
+			'site_transient_update_plugins',
+			function ( $transient ) use ( $container ) {
 
-			if ( isset( $release->new_version ) && version_compare( $release->new_version, $plugin->version(), '>' ) ) {
-				$transient->response[ $plugin->basename() ] = $release;
-			}
+				/**
+				 * The plugin instance.
+				 *
+				 * @var Plugin $plugin
+				 */
+				$plugin  = $container['plugin'];
+				$release = $container['get_release_data'];
 
-			return $transient;
-		} );
+				if ( isset( $release->new_version ) && version_compare( $release->new_version, $plugin->version(), '>' ) ) {
+					$transient->response[ $plugin->basename() ] = $release;
+				} else {
+					$transient->no_update[ $plugin->basename() ] = (object) array(
+						'id'            => $plugin->basename(),
+						'slug'          => $plugin->slug(),
+						'plugin'        => $plugin->basename(),
+						'new_version'   => $plugin->version(),
+						'url'           => $plugin->uri(),
+						'package'       => '',
+						'icons'         => array(),
+						'banners'       => array(),
+						'banners_rtl'   => array(),
+						'tested'        => $plugin->requires_wp_version(),
```

Comment:
Just checking that this is the right value. Tested up to and the minimum version requirement seem like they would be different. I don't know that this will be displayed anywhere, but could result in saying the plugin has not been tested against recent versions.